definitions/npm/redux_v3.x.x/flow_v0.28.x-v0.32.x/redux_v3.x.x.js

```diff
@@ -0,0 +1,54 @@
+declare module 'redux' {
+
+  /*
+
+    S = State
+    A = Action
+
+  */
+
+  declare type Dispatch<A: { type: $Subtype<string> }> = (action: A) => A;
```

Comment:
@pcardune I asked [the same question](https://github.com/reactjs/redux/pull/1887#issuecomment-241446828) and was pointed [here](https://github.com/agentcooper/react-redux-flow-example/blob/master/src/types/index.js#L20) for the solution.